example_buffer.py

```diff
@@ -132,10 +150,69 @@ def _rsync_dir(source_dir, dest_dir):
                         stderr=rsync_log)
 
 
-def fill_and_wait(bufsize=dual_net.EXAMPLES_PER_GENERATION,
+def _determine_chunk_to_make(write_dir):
+    models = fsdb.get_models()
+    # Last model is N.  N+1 is training.  We should gather games for N+2.
+    chunk_to_make = os.path.join(write_dir, str(
+        models[-1][0] + 2) + '.tfrecord.zz')
+    while tf.gfile.Exists(chunk_to_make):
```

Comment:
It's in fill_and_wait_time(), not the chunk-to-make function.  I agree, otherwise it would feel weird.

at the moment, though, this gatherer is the only real centralized point that knows the counts of things, so it definitely has to be somewhere in here...